packages/renovate-config-monorepo/generate.js

```diff
@@ -48,10 +48,11 @@ const repoGroups = {
   flopflip: 'https://github.com/tdeekens/flopflip',
   framework7: 'https://github.com/framework7io/framework7',
   gatsby: 'https://github.com/gatsbyjs/gatsby',
-  graphqlcodegenerator: [
+  'graphql-code-generator': [
```

Comment:
Changing this existing name will cause 10s of thousands of config error PRs, because it's referenced as a preset from elsewhere. So please don't do that in this PR and let's do it separately.